### PR TITLE
Release Label creation automation

### PR DESCRIPTION
**Pull request by prudhvigodithi**
### Description
Release Label creation automation

### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/3661

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by prudhvigodithi** [comment]
Screenshot 2023-06-22 at 3 05 12 PM
Screenshot 2023-06-22 at 3 04 44 PM

File: .github/workflows/os-increment-plugin-versions.yml [hunk 1]
```diff
@@ -85,6 +85,36 @@ jobs:
...
+          result-encoding: string
+          script: |
+            const { data: labels } = await github.rest.issues.listLabelsForRepo({
+              owner: context.repo.owner,
+              repo: "${{ matrix.entry.repo }}",
+              per_page: 100
+            });
+            const labelFound = labels.some(label => label.name === 'v${{ env.OPENSEARCH_VERSION_NUMBER }}');
+            console.log(labelFound);
+            return labelFound
```
Comment (gaiksaya) [inline .github/workflows/os-increment-plugin-versions.yml:102]: Interesting, so this acts as a function in github actions?

**Review by gaiksaya — APPROVED** [review]
This is amazing! Thanks @prudhvigodithi 

Once we know this works, can you remove the step to create labels from release template?
Thanks!

File: .github/workflows/os-increment-plugin-versions.yml (hunk 1 above)
Comment (prudhvigodithi) [reply .github/workflows/os-increment-plugin-versions.yml:102]: Yes Sayali Once works I will update the document
Also ya this is equivalent to `set-output` https://github.com/actions/github-script#reading-step-results

**Pull request merged**